recipe/install_airflow.sh

```diff
@@ -2,5 +2,12 @@
 
 set -xe
 
-$PYTHON setup.py compile_assets
-$PYTHON -m pip install --no-deps --ignore-installed . --verbose
+# dev/hatch_build.py was purposefully removed from the release but we need it
+mkdir -p dev
+for file in hatch_build.py airflow_pre_installed_providers.txt
+do
+  curl -SL "https://raw.githubusercontent.com/apache/airflow/${PKG_VERSION}/dev/${file}" \
+      > "dev/${file}"
+done
```

Comment:
This is obviously not ideal but the `dev` directory is explicitly being excluded from the source code and we seem to need it.